packages/forma-36-website/src/pages/components/modal/index.mdx

```diff
@@ -164,4 +165,51 @@ function ModalExample() {
 - To keep UX copy actionable use present tense for user interactions
 - Avoid “never,” “always” and other absolutes
 - Use contractions when possible
-- Avoid exclamation marks
\ No newline at end of file
+- Avoid exclamation marks
+
+## Using the modal launcher <a name="using-the-modal-launcher" />
+
+The `ModalLauncher.open` utility can be used to wrap any custom result, based on a user's interaction with a Modal component, in a promise and await it.
```

Comment:
Let's remove this for now and instead mark the component as alpha (we don't need in-depth docs for alpha components).

Reply:
done